MAINTAINERS.md

```diff
@@ -1,11 +1,18 @@
-# Observability Maintainers
+## Overview
 
-## Maintainers
-| Maintainer    | GitHub ID                                       | Affiliation |
-|---------------|-------------------------------------------------|-------------|
-| David Cui     | [davidcui1225](https://github.com/davidcui1225) | Amazon      |
-| Eric Wei      | [mengweieric](https://github.com/mengweieric)   | Amazon      |
-| Joshua Li     | [joshuali925](https://github.com/joshuali925)   | Amazon      |
-| Shenoy Pratik | [ps48](https://github.com/ps48)                 | Amazon      |
-| Kavitha Mohan | [kavithacm] (https://github.com/kavithacm)      | Amazon      |
-| Eugene Lee    | [eugenesk24] (https://github.com/eugenesk24)    | Amazon      |
\ No newline at end of file
+This document contains a list of maintainers in this repo. See [opensearch-project/.github/RESPONSIBILITIES.md](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#maintainer-responsibilities) that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Current Maintainers
+
+| Maintainer        | GitHub ID                                         | Affiliation |
+| ----------------- | ------------------------------------------------- | ----------- |
+| David Cui         | [davidcui1225](https://github.com/davidcui1225)   | Amazon      |
```

Comment:
What is the workflow you are working? Am wondering if we can hop on a call... I think it is best to avoid including previous commits into your own PR from other folks

Reply:
I branched from 2.x & cherry picked relevant files from main branch fork..

Reply:
Doesn’t it represents the latest correct maintainers info ?